submissions/israelOladosu/php-db/classes/queryDb.class.php

```diff
@@ -30,4 +30,20 @@ public function getAllCustomers()
         }
     }
 
-};
\ No newline at end of file
+    public function getOrders($id)
+    {
+        try {
+
+            $sql = 'SELECT orders.id, items.quantity, items.unit_price, items.total_amount, orders.made_at, orders.delivery_address, products.product_name FROM items LEFT JOIN orders ON orders.customer_id = items.order_id LEFT JOIN products ON products.id = items.product_id WHERE orders.customer_id = ?';
```

Comment:
kindly break this line of code

Reply:
Also, why passing `?` in this statement `WHERE orders.customer_id = ?`

Reply:
Also, is this how `orders.customer_id = items.order_id`  relation between `orders` and `items` tables exist? I think it suppose to be `orders.id = items.order_id`.  Then you can get the customer_id on the `orders` table. That is now a relationship between the `orders` and the `addresses` or `customers` table

Reply:
> Also, why passing `?` in this statement `WHERE orders.customer_id = ?`

no reason in particular, just wanted to try out the use of placeholders as we were taught in the lesson.

Reply:
> Also, is this how `orders.customer_id = items.order_id` relation between `orders` and `items` tables exist? I think it suppose to be `orders.id = items.order_id`. Then you can get the customer_id on the `orders` table. That is now a relationship between the `orders` and the `addresses` or `customers` table

Right, Thanks for the correction.

Reply:
> > Also, why passing `?` in this statement `WHERE orders.customer_id = ?`
> 
> no reason in particular, just wanted to try out the use of placeholders as we were taught in the lesson.

ok. You can leave it then